instrumentation/github.com/aws/aws-sdk-go-v2/otelaws/example/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+version: "3.7"
+services:
+  aws-sdk-client:
+    build:
+      dockerfile: $PWD/Dockerfile
```

Comment:
Since in the instruction you mentioned use can run `docker-compose` command from `example` dir I think here we can remove `$PWD` from dockerfile path and use `.` in context as well.

Reply:
$PWD is more robust.